contracts/governance/extensions/GovernorTimelockAccess.sol

```diff
@@ -17,16 +17,18 @@ import {Time} from "../../utils/types/Time.sol";
  */
 abstract contract GovernorTimelockAccess is Governor {
     struct ExecutionPlan {
+        uint16 length;
         uint32 delay;
-        OperationDetails[] operations;
+        // We use mappings instead of arrays because it allows us to pack values in storage more tightly without storing
+        // the length redundantly.
+        // We pack 8 operations' data in each bucket. Each uint32 value is set to 1 upon proposal creation if it has to
+        // be scheduled and relayed through the manager. Upon queuing, the value is set to nonce + 1, where the nonce is
+        // that which we get back from the manager when scheduling the operation.
+        mapping (uint256 bucket => uint32[8]) managerData;
```

Comment:
I'd agree with `bucket` if this was a generic library, but it can be contextualized:

```suggestion
        mapping (uint256 operationId => uint32[8]) managerData;
```

Thoughts?